testing/xfpga/test_enum_c.cpp

```diff
@@ -433,7 +442,16 @@ TEST_P(enum_c_p, device_id) {
   EXPECT_EQ(
       xfpga_fpgaEnumerate(&filter_, 1, tokens_.data(), tokens_.size(), &num_matches_),
       FPGA_OK);
-  EXPECT_EQ(num_matches_, platform_.devices.size() * 2);
+  EXPECT_EQ(num_matches_, platform_.devices.size() * 2 - device.num_vfs);
+
+  for (int i = 0; i < device.num_vfs; ++i) {
+    num_matches_ = 0;
+    ASSERT_EQ(fpgaPropertiesSetDeviceID(filter_, device.device_id+i), FPGA_OK);
+    EXPECT_EQ(xfpga_fpgaEnumerate(&filter_, 1, tokens_.data(), tokens_.size(),
```

Comment:
Isn't this loop bleeding tokens?  ie, each time through tokens_.data() is set to a cloned token.  Each iteration after the first will leak the token.

Reply:
miss this one?